.github/workflows/release-python.yml

```diff
@@ -0,0 +1,115 @@
+name: Release Python
+
+on:
+  push:
+    tags:
+      - python/v*
+
+jobs:
+  macos:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
+      - name: Build
+        run: cargo build --release
+      - name: Build wheels - x86_64
+        uses: messense/maturin-action@v1
+        with:
+          target: x86_64
+          args: -i python --release --manifest-path crates/python/Cargo.toml --out dist --sdist
+      - name: Install wheel - x86_64
+        run: |
+          pip install qcs-sdk-python --no-index --find-links dist --force-reinstall
+      - name: Build wheels - universal2
+        if: ${{ matrix.python-version >= '3.8' || matrix.python-version == '3.10' }}
```

Comment:
Frankly, this could be `!= 3.7` instead. then above could then be `== 3.7`, since we don't need both x86_64 and universal2 wheels for a given Python version.

Reply:
I will update the condition, thanks for the suggestion. 

For the `universal2` wheels. `pip` started supporting them in `20.3` so I was building them for compatibilities sake. `20.3` is coming up on 2 years old now, so I can see an argument for them not being relevant anymore. That said, they were easy enough to build so I left it in.

Reply:
I'm confused what you are saying. I _think_ you mean that you see an argument for `x86_64`-only not being relevant anymore?

My primary reason for suggesting doing only `universal2` is to decrease CI time, since `macos` (and `windows`) cost more than `linux` (not sure if we're paying for CI or getting it free).

Reply:
Let me clarify. There are two reasons to build for both `x86_64` and `universal2`:

1) `universal2` is basically just a `x86_64` and `arm64` binary in one, meaning it's twice the size.

2) Since `universal2` is relatively new, it's only supported on `pip >= 20.3`. If we don't build x86_64 wheels, then users on a previous version will have to update `pip`.

What I was trying to say is that you could argue against (2) since `pip` 20.3 is almost 2 years old now.

In short the tradeoff is either:

Support smaller binaries and older versions of pip for `x86_64` users OR decrease CI time.

I'll go ahead and skip building `x86_64` wheels for >= `3.8`. It's an experience improvement that no one may care about. If someone raises it, we can easily add it back in.